data/data-pipeline/data_pipeline/tests/test_score_calculation.py

```diff
@@ -0,0 +1,286 @@
+from ast import PyCF_TYPE_COMMENTS
+from pathlib import Path
+import pandas as pd
+import pytest
+import data_pipeline.score.field_names as field_names
+import data_pipeline.score.field_names as field_names
+from data_pipeline.utils import get_module_logger
+from data_pipeline.config import settings
+from data_pipeline.score.score_narwhal import ScoreNarwhal
+
+logger = get_module_logger(__name__)
+
+
+@pytest.fixture
+def donut_hole_adjacency_parameters():
+    """Test that donut indicator uses the right threshold"""
+    return (
+        # threshold boolean
+        field_names.ADJACENT_TRACT_SCORE_ABOVE_DONUT_THRESHOLD,
+        # percentile equivalent column
+        field_names.SCORE_N_COMMUNITIES + field_names.ADJACENCY_INDEX_SUFFIX,
+        # threshold
+        ScoreNarwhal(final_score_df).SCORE_THRESHOLD_DONUT,
+    )
+
+
+@pytest.fixture
+def donut_hole_income_threshold_and_column():
+    """Test that donut income uses the right threshold"""
+    return (
+        # threshold
+        ScoreNarwhal(final_score_df).LOW_INCOME_THRESHOLD_DONUT,
+        field_names.FPL_200_SERIES_IMPUTED_AND_ADJUSTED_DONUTS,
+    )
+
+
+@pytest.fixture
+def final_score_df():
```

Comment:
I'm guessing this is slow too, and I think is duplicate below. This is a nit but we can probably factor some of these into a common fixtures.py

Reply:
yeah that sounds good to me!